docs/backend/timezone-handling.md

```diff
@@ -0,0 +1,85 @@
+# Overview
+
+The BCIERS application follows a **UTC-first** approach for all timestamp storage and processing. This ensures consistency, prevents timezone-related bugs, and follows international standards for database design.
+
+## Backend (Django) Configuration
+
+### Settings
+
+The Django application is configured in `bc_obps/bc_obps/settings.py`:
+
+```python
+TIME_ZONE = "UTC"
+USE_TZ = True
+```
+
+### Key Implications
+
+- `timezone.now()` returns UTC timestamps
+- All `DateTimeField` values are timezone-aware and stored in UTC
+- Django automatically handles timezone conversions when needed
+
+### Best Practices
+
+#### Always Use Timezone-Aware Datetimes
+
+```python
+# ✅ Good - timezone-aware
+from django.utils import timezone
+current_time = timezone.now()
+
+# ❌ Bad - naive datetime
+from datetime import datetime
+current_time = datetime.now()
+```
+
+## Frontend Handling
+
+### Timezone Conversion
+
+The frontend converts UTC timestamps to the user's local timezone (typically Pacific Time for BC users) for display:
+
+```typescript
+// From bciers/libs/utils/src/formatTimestamp.ts
+const formatTimestamp = (timestamp: string) => {
+  const date = new Date(timestamp).toLocaleString("en-CA", {
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+    timeZone: "America/Vancouver",
+  });
+
+  const timeWithTimeZone = new Date(timestamp).toLocaleString("en-CA", {
+    hour: "numeric",
+    minute: "numeric",
+    timeZoneName: "short",
+    timeZone: "America/Vancouver",
+  });
+
+  return `${date}\n${timeWithTimeZone}`;
+};
+```
+
+## Querying Database
+
+When querying the database directly (e.g., in Metabase, SQL queries, debugging):
+
+1. **Remember you're seeing UTC times**: A timestamp showing `2024-01-15 22:00:00` is 10:00 PM UTC, which is 2:00 PM Pacific Time
```

Comment:
Is UTC 22:00 always 2pm pacific? Ie, does UTC have daylight savings on the same schedule as us? If not, maybe we could update this to `2:00 PM Pacific Standard (or Daylight, whichever it is) Time`

Reply:
Good call, Thanks! 👍